### PR TITLE
NSKOPS-116 dependencies have been modified

### PR DESCRIPTION
**Pull request by ghostik**
The dependence of this plugin was changed from 0.90.11 to 1.7.1.
---------------------------------------------------------------------------
    <groupId>org.elasticsearch</groupId>
    <artifactId>elasticsearch-srv-discovery</artifactId>
    <version>1.7.1</version>
    <packaging>jar</packaging>
---------------------------------------------------------------------------
    <properties>
        <elasticsearch.version>1.7.1</elasticsearch.version>
    </properties>
--------------------------------------------------------------------------

### CODE REVIEW

**Comment by ghostik** [comment]
@mkostrikin please review

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.elasticsearch</groupId>
     <artifactId>elasticsearch-srv-discovery</artifactId>
-    <version>0.90.11-8</version>
+    <version>1.7.1</version>
```
Comment (mkostrikin) [inline pom.xml]: As far as I understand, this code will build a deb package with ES plugin with version 1.7.1 and for ES 1.7.1
How 0.90.11-8 should be build after merge?

Comment (ghostik) [inline pom.xml]: As a result of the build will be created a plugin with new version 1.7.1 and this action has no any impact on the previous deb packet, I think.

Comment (mkostrikin) [inline pom.xml]: @sorenmat how we should behave regard keeping 0.90 maintainable and build 1.7 as well?
Should we consider about using maven profiles ?

Comment (sorenmat) [inline pom.xml]: I would just create a branch of master with the 0.90 code in it and call it 0.90. And then do your work, and merge that to master. 
We would like to upgrade everything to a newer ES version anyways. And we do have the current 0.90 as a binary.

File: src/main/java/org/elasticsearch/discovery/srv/SrvDiscovery.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015 Grant Rodgers
+ * Copyright (c) 2015 GitHub
```
Comment (mkostrikin) [inline src/main/java/org/elasticsearch/discovery/srv/SrvDiscovery.java]: Why this was changed?

File: src/main/java/org/elasticsearch/discovery/srv/SrvDiscovery.java [hunk 3]
```diff
@@ -22,25 +22,31 @@
...
 import org.elasticsearch.discovery.zen.ZenDiscovery;
+import org.elasticsearch.discovery.zen.elect.ElectMasterService;
 import org.elasticsearch.discovery.zen.ping.ZenPingService;
 import org.elasticsearch.node.settings.NodeSettingsService;
 import org.elasticsearch.threadpool.ThreadPool;
 import org.elasticsearch.transport.TransportService;
 
+/**
+ *
+ */
```
Comment (mkostrikin) [inline src/main/java/org/elasticsearch/discovery/srv/SrvDiscovery.java]: What a mystery starts?

**Comment by mkostrikin** [comment]
PR must be against branch 1.7.1
please fix and recreate

File: src/main/java/org/elasticsearch/discovery/srv/SrvDiscovery.java (hunk 2 above)
Comment (ghostik) [inline src/main/java/org/elasticsearch/discovery/srv/SrvDiscovery.java]: I don't change this code. I replace the old SrvDiscovery.java ( from TS repo) to current version SrvDiscovery.java from GitHub.

File: src/main/java/org/elasticsearch/discovery/srv/SrvDiscovery.java (hunk 3 above)
Comment (ghostik) [inline src/main/java/org/elasticsearch/discovery/srv/SrvDiscovery.java]: What do you mean?

Comment (mkostrikin) [inline src/main/java/org/elasticsearch/discovery/srv/SrvDiscovery.java]: What is a purpose of this three lines of code?

**Pull request closed without merging**